packages/core/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "../../node_modules/ng-packagr/package.schema.json",
+  "name": "@agm/core",
+  "description": "Angular components for Google Maps",
+  "author": "Sebastian Holstein <info@sebastian-holstein.de>",
+  "keywords": [
+    "angular",
+    "ng",
+    "google-maps",
+    "maps",
+    "google",
+    "gmaps",
+    "angular2"
+  ],
+  "peerDependencies": {
+    "@angular/common": "^6.0.0 || ^7.0.0",
+    "@angular/core": "^6.0.0 || ^7.0.0"
+  },
+  "ngPackage": {
+    "lib": {
+      "entryFile": "./index.ts",
+      "umdId": "ngmaps.core"
+    },
+    "dest": "../../dist/@agm/core",
+    "workingDirectory": "../../.tmp"
```

Comment:
hey, @terencehonles . You are using an old version of ng-packager, which is not backwards compatible. See https://github.com/ng-packagr/ng-packagr/blob/master/CHANGELOG.md#breaking-changes-5
Can you update it to match latest lib versions?

Reply:
You probably meant to comment on #1634 , but this is just a change in ng-packagr's schema. I had definitely used a 5x version of ng-packagr and must have used some stale docs to set it up. I fixed the change in #1634, but this PR will probably need some more time to verify it's still working w/ the latest changes from master.

Reply:
If you noticed, I created a PR to your PR and smoke tested it. Check it out